plugins/modules/cargo.py

```diff
@@ -106,6 +115,11 @@
   community.general.cargo:
     name: ludusavi
     directory: /path/to/ludusavi/source
+
+- name: Install "serpl" Rust package with ast_grep feature
+  community.general.cargo:
+    name: serpl
+    features: [ast_grep]
```

Comment:
Why not have a proper YAML list?
```suggestion
    features:
      - ast_grep
```

Reply:
The origins of this is because I startem with ast_grep `str`. I think it depends on the preference. I even thought of doing some module logic so when there is just single `str` passed it automatically converts it into single item list.

Also when I specify `cargo install serpl --features ast_grep` it's just on the single line just as `serpl` name.

If that's more idiomatic way to describe it in YAML feel free to to edit. I would just use this syntax for single item array, otherwise:
```yaml
features:
  - one
  - two
```

Looks better to me.

Reply:
Okay I looked it up. This is JSON sequence syntax that YAML understands. +1 for changing this, I can commit it later as I'm AFK.

Reply:
Ah sorry I didn't see `Commit suggestion` on mobile app. I see it's available in the desktop.